GiniVision/Classes/CameraContainerViewController.swift

```diff
@@ -48,17 +48,16 @@ internal class CameraContainerViewController: UIViewController, ContainerViewCon
                 DispatchQueue.main.async {
                     self.navigationController?.pushViewController(viewController, animated: true)
                 }
-                
-        }, failureBlock: {[unowned self] error in
-            switch error {
-            case CameraError.notAuthorizedToUseDevice:
-                print("GiniVision: Camera authorization denied.")
-            case is DocumentValidationError:
-                self.showNotValidDocumentError()
-            default:
-                print("GiniVision: Unknown error when using camera.")
-            }
-        })
+            }, failureBlock: {[unowned self] error in
+                switch error {
+                case CameraError.notAuthorizedToUseDevice:
+                    print("GiniVision: Camera authorization denied.")
+                case is DocumentValidationError:
+                    self.showNotValidDocumentError(error: error as! DocumentValidationError)
```

Comment:
We should show the validation errors in the CameraController so that it is shown the same way with the Component API, too. (Like the photo library permission alert)

Reply:
Yes, I don't why I did it well for the Photo library permissions denied dialog and not for this.
Thanks 👍